src/robotide/parserlog/parserlog.py

```diff
@@ -125,7 +119,6 @@ def __init__(self, notebook, log):
 
     def _add_to_notebook(self, notebook):
         notebook.add_tab(self, 'Parser Log', allow_closing=True)
-        notebook.show_tab(self)
```

Comment:
This is needed to show Parser Tab when selected from Menu.

Reply:
Yeah, but that's what was actually causing the error. It occurs when it tries to switch to Parser Tab at startup. I guess I could move it... to avoid error and show Parser Tab when selected from Menu.